oauth2/authorization_code/README.md

```
@@ -0,0 +1,248 @@
+  authorization_code.WithSessionHooks(<ISessionHooks implementation>),      // used for storing/retrieving tokens
+  authorization_code.WithOffline(),                     // optional - include if you'd like to maintain refresh tokens and a long session
+  authorization_code.WithTokenValidation(
+    true,                                               // will validate token signature via JWKS
+    jwt.WillValidateAlgorithm(),                        // will validate the token alg is RS256
+  ),
+)
+```
+
+### Available Methods
+
+### Available Methods
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)